source/optional_features/custom_metadata/index.html.md.erb

```diff
@@ -0,0 +1,50 @@
+---
+title: Custom metadata
+weight: 135
+---
+
+# Custom metadata
+
+You can add custom metadata to a new payment. For example, you can add a reference number from your finance or accounting system, so you can reconcile the payment against bank transaction information.
+
+You can add metadata when you make an API call to create a new payment. You cannot add metadata to Direct Debit transactions or payment links.
```

Comment:
worth saying "you cannot currently add"..? - although i know this breaks the rules around temporality. we do have plans to extend to these features in future.

Reply:
I think would break the temporality rule as you say - sorry!

Reply:
no worries!